libcnb-data/src/launch.rs

```diff
@@ -155,6 +195,7 @@ impl ProcessBuilder {
                 args: Vec::new(),
                 direct: false,
                 default: false,
+                working_directory: WorkingDirectory::App,
```

Comment:
The rustdocs for `ProcessBuilder` needs updating to mention this default value (like it does for `direct`, `default` etc).